docs/api/en/objects/BatchedMesh.html

```diff
@@ -222,22 +228,35 @@ <h3>
 			the length of the given geometry index buffer.
 		</p>
 		<p>
-			Adds the given geometry to the [name] and returns the associated index referring to it.
+			Adds the given geometry and a new instance to the [name] and returns the associated index referring to it to be used by other functions.
+		</p>
+
+		<h3>
+			[method:Integer addInstance]( [param:Integer id] )
+		</h3>
+		<p>
+			[page:Integer id]: The id of a previously added item via "addGeometry" or "addInstance" to add into the [name] to render. 
+		</p>
+		<p>
+			Adds a a new instance to the [name] using the same geometry from the given id and returns the associated index referring to it to be used
+			by other functions.
 		</p>
 
 		<h3>
 			[method:Integer setGeometryAt]( [param:Integer index], [param:BufferGeometry geometry] )
 		</h3>
 		<p>
-			[page:Integer index]: Which geometry index to replace with this geometry.
+			[page:Integer index]: Which item index to replace with this geometry.
```

Comment:
Wouldn't "geometry" be the right term in this case? Questions:

1. We're using the terms "index" and "id" interchangeably in this section, using "id" consistently might be best.
2. When calling `setGeometryAt` for an ID previously allocated with `addGeometry`, are instances of that geometry affected?
3. When calling `setGeometryAt` for an ID previously allocated with `addInstance`, is any other instance affected? 
4. Is an "instance of an instance" different in any way from an instance created from the same base geometry?

We could consider not supporting creation of "instances of instances" quite yet, if the preferred answers to these questions are non-obvious.